src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -160,7 +162,7 @@
         </div>
       {% endif %}
       {% for customization in product.customizations.textCustomizations %}
-        <p><strong>{{ customization.name }}</strong> {{ customization.value }}</p>
+      <p><strong>{{ '%name%:'|trans({'%name%': customization.name}, 'Admin.Ordercustomers.Feature') }} </strong> {{ customization.value }}</p>
```

Comment:
That's weird this data is supposed to be translated in the database (in the `customization_field_lang`) I guess this can be useful for our fixtures data (and only them), but they'd rather be correctly inserted in the database rather than rely on the Translator
Well it will probably work anyway, but still a weird use

Reply:
@Progi1984 did a review on this because we needed to manage the `:` spacing between EN and other languages

Reply:
Oooh I get it! Smart!